west.yml

```diff
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: soburi
+      url-base: https://github.com/soburi
```

Comment:
we usually do not pull directly from outside repos. See the other modules in this file.

Reply:
I need to submit it to zephyr as the new module.
I will update it after approve it.